.github/workflows/multibuild.yaml

```diff
@@ -363,6 +340,40 @@ jobs:
       - name: Generate SHA256 checksums
         working-directory: tarballs
         run: sha256sum * > checksums.txt
+      - name: Update SHA256 checksums in universal.sh
```

Comment:
We have a chicken and egg problem now @cpswan (with the way jobs are currently laid out):

Universal.sh needs to be updated with the new checksums (as I've added checksum verification), but this then requires us to regenerate the checksums for universal.sh before provenance occurs.

We'll have the same problem with the rest of our manifests if we decide to upload them to the release (which I think we've already discussed doing).